src/Components/Color/Color.jsx

```diff
@@ -1,8 +1,10 @@
 import { useState } from "react";
 import "./Color.css";
+import ColorForm from "../ColorForm/ColorForm";
 
-export default function Color({ color, onDelete }) {
+export default function Color({ color, onDelete, onUpdate }) {
```

Comment:
Consider changing the naming of Color to ColorCard? As well as you Color.jsx. I had the same before but changed it to make it more clear.

Reply:
Thanks for the tip! You’re right, it makes it clearer :)